README.md

```diff
@@ -106,4 +126,25 @@ This option allows control over the alignment of the image on the page.
 
 This option allows you to select a particular page from a draw.io file to
 create the image from. Note: an invalid page-index will revert to one of the
-other valid pages (draw.io binary functionality)
\ No newline at end of file
+other valid pages (draw.io binary functionality)
+
+### Scale
+- *Formal Name*: `:scale:`
+- *Default Value*: `drawio_default_scale` set in conf.py
+- *Possible Values*: any positive integer
+
+This simply scales the size of the output image. By default draw.io *usually* 
```

Comment:
Please mention this with numbers (e.g. a nominally 50x50 image will become approximately 100x100 with a scale of 2)